kolibri/plugins/learn/assets/src/views/HybridLearningContentCardListView.vue

```diff
@@ -32,7 +32,11 @@
           class="metadata-info"
           :style="{ color: $themePalette.grey.v_700 }"
         >
-          <LearningActivityLabel :contentNode="content" class="learning-activity-label" />
+          <LearningActivityLabel
+            :contentNode="content"
+            labelAfter
```

Comment:
for consistency in the code, would you  mind to change https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue#L22 to use the same syntax?

Reply:
Thanks, @jredrejo. I'm sorry for the late reply. I checked how we use Boolean props elsewhere, and it seems that across the whole codebase, we use explicit `true` value for many other boolean props, so even if I removed Boolean value from your reference, we'd still have many other places (just search for  `="true"` ) so it wouldn't probably help much overall with consistency. I don't feel strongly opinionated about these two syntaxes. Did you consider syntax without the Boolean value confusing? I could also use explicit `:labelAfter="true"` instead.

Reply:
Specs don't have a favourite setting, https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute. From a personal point, I prefer the presence/absence of the attribute to reflect its value. It looks more ellegant and readable for me, but it's a completely personal preference.
In any case, I think it's important to have consistency in the code. For this attribute we're using it only in one more place, so whatever is decided is good for me is it's the same in both parts.

Reply:
Thanks. I appreciate its simplicity too. On the other hand, the advantage of having it explicitly defined might be that it's clear that it's one of Vue's `prop`s.

In any case, I'd say it's a matter of all Boolean attributes rather than this one specific attribute, and since we mostly use explicit syntax across the whole codebase, I think I'll use that.  I'm going to sneak the update in one of my next PRs.

Reply:
Addressed here https://github.com/learningequality/kolibri/pull/8891